app/Console/Commands/MakeSourceSms.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Signup;
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class MakeSourceSms extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:smssource';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change all "sms-mobilecommons" source value to "sms" in the signups and posts tables';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Start updating signups
+        info('rogue:smssource: Starting to update signups!');
+
+        // Get all signups that have "sms-mobilecommons" set as their source
+        $query = Signup::where('source', 'sms-mobilecommons');
+
+        // Update all the signups
+        $query->chunkById(100, function ($signups) {
```

Comment:
There are about 90-100k records that need to be changed. Should I increase this?

Reply:
Great question! On the quantity script I just changed it from 100 to 1000 but we haven't tried it yet. There are A LOT of signups so 1000 makes sense to me

Reply:
Also @chloealee 100% up to you, but this line and `$query = Signup::where('source', 'sms-mobilecommons');` can be condensed into one line like 
```
Signup::where('source', 'sms-mobilecommons')->chunkById(100, function ($signups) {
``` 
but totally go with whatever you find more readable.

Reply:
oooo I like that a lot more - thanks @katiecrane !